index.html

```diff
@@ -32,12 +32,154 @@ <h1> Charlie
     <button class="collab">
       Start collaboration
     </button>
-    
+    <div class="image22">
+      <object class="image2" data="./svgfiles/image_2.svg" type="image/svg+xml" ></object>
+    </div>
   </section>
 
-  <div class="image22">
-    <object class="image2" data="./svgfiles/image_2.svg" type="image/svg+xml" ></object>
-  </div>
  
+
+  <section class="projects">
+    <div class="projects-title"><h2>Projects</h2></div>
+    <div class="projects-1">
+      <div class="projects-1_img">
+        <object  data="./svgfiles/Rectangle 1.svg" type="image/svg+xml" ></object>
+      </div>
+      <div class="project-1_name">
+        <h2>Project name goes here</h2>
+        <div class="project-1_name_lang">
+          <div class="langs">HTML/CSS</div>
+          <div class="langs-1">Ruby on Rails</div>
+          <div class="langs-2">JavaScript</div>
+        </div>
+        <button class="see-pro">See this project &rarr;</button>
+      </div>
+  </div>
+
+    <div class="projects-2">
+      <div class="projects-2_img">
+         <object  data="./svgfiles/Rectangle 3.svg" type="image/svg+xml" ></object>
+       </div>
+      <div class="project-2_name">
+          <h2>Project name goes here</h2>
+          <div class="project-1_name_lang">
+          <div class="langs">HTML/CSS</div>
+          <div class="langs-1">Ruby on Rails</div>
+          <div class="langs-2">JavaScript</div>
+        </div>
+          <button class="see-pro">See this project &rarr;</button>
+      </div>
+    </div>
+  
+    <div class="projects-3">
+      <div class="projects-3_img">
+      <object  data="./svgfiles/Rectangle 2.svg" type="image/svg+xml" ></object>
+      </div>
+      <div class="project-3_name">
+        <h2>Project name goes here</h2>
+        <div class="project-1_name_lang">
+        <div class="langs">HTML/CSS</div>
+        <div class="langs-1">Ruby on Rails</div>
+        <div class="langs-2">JavaScript</div>
```

Comment:
- [ ] I think it would be better to list the project technologies (e.g., "JavasScript", "HTML" and "CSS") in a list ```ul``` and each of them a ```li``` 😎

    using the ```<ul>``` tag to create unordered lists in your HTML can help improve the semantic structure, accessibility, and maintainability of your page. 👍